spec/GameSpec.js

```diff
@@ -0,0 +1,36 @@
+describe("Game", function() {
+  var game;
+
+  beforeEach(function() {
+    game = new Game();
+  });
+
+  it('should have a property player1', function() {
+    expect(game.player1).not.toBeUndefined();
+  });
+
+  it('should have a property player2', function() {
+    expect(game.player2).not.toBeUndefined();
+  });
+
+  it('should have a property gameBoard', function() {
+    expect(game.gameBoard).not.toBeUndefined();
+  });
+
+  it('should have an initialization method that returns the Game object', function() {
+    expect(game.init()).toEqual(game);
+  });
+
+  it('should have an assignSides method that returns the Game object', function() {
+    expect(game.assignSides()).toEqual(game);
+  });
+
+  it('should assign the player1 object a side of 1', function() {
+    expect(game.player1.side).toEqual(1);
```

Comment:
similarly, game.getSide(); .. either player1 is an argument of getSide(player1) or this isn't even something that is accessible via a method and the game keeps track of it internally and passes it to objects (player1 or player2) that require it to make a move.